Documentation/ColumnsConfig/Type/Category/Properties/TreeConfig.rst

```diff
@@ -30,8 +30,9 @@ treeConfig
       The setting takes a CSV value, e.g. `2,3,4711`, which takes records of the uids
       `2`, `3` and `4711` into account and creates a tree of these records.
 
-      Additionally, each value used in `startingPoints` may be fed from a site
-      configuration by using the `###SITE:###` syntax.
+      Additionally, each value used in `startingPoints` may be fed from a
+      :ref:`site configuration <t3coreapi:sitehandling-inTypoScript>`
```

Comment:
IMHO this is not about TypoScript, but about site handling in general.

Reply:
There is no dedicated section about custom properties in site configuration, so I thought it would at least be helpful to see how site configuration is used in other places.

Reply:
Do you know of  a better link target?
this was mine: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/SiteHandling/UseSiteInTypoScript.html

Reply:
Just the start page: https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/SiteHandling/Index.html

You link "site configuration" and that chapter is about site configuration.